app/src/main/java/com/example/stromtracker/ui/haushalt/HaushaltViewModel.kt

```diff
@@ -1,9 +1,28 @@
 package com.example.stromtracker.ui.haushalt
 
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.example.stromtracker.database.DataRepository
+import com.example.stromtracker.database.Haushalt
+import com.example.stromtracker.database.Kategorie
 
-class HaushaltViewModel : ViewModel() {
+class HaushaltViewModel(application:Application) : AndroidViewModel(application) {
```

Comment:
What's the reason to change to an AndroidViewModel instead of a ViewModel?

Reply:
That was my decision, in order to instantiate the database we need the application context, which we get with AndroidViewModel.